client-linuxapp/src/main.rs

```diff
@@ -145,15 +152,28 @@ async fn get_to1d_from_rv(devcred: &dyn DeviceCredential) -> Result<COSESign> {
         bail!("No rendezvous information found that's usable for the device");
     }
     log::trace!("Rendezvous info: {:?}", rv_info);
+    Ok(rv_info)
+}
 
-    let mut client = get_client(rv_info)
-        .await
-        .context("Error getting usable rendezvous client")?;
-    log::trace!("Got a usable client: {:?}", client);
-
-    perform_to1(devcred, &mut client)
-        .await
-        .context("Error performing TO1")
+async fn get_to1d(
+    devcred: &dyn DeviceCredential,
+    mut client_list: Vec<ServiceClient>,
+) -> Result<COSESign> {
+    for client in client_list.as_mut_slice() {
+        match perform_to1(devcred, client)
+            .await
+            .context("Error performing TO1")
+        {
+            Ok(to1) => {
+                return Ok(to1);
+            }
+            Err(e) => {
+                log::trace!("{} with {:?}", e, client);
+                continue;
+            }
+        }
+    }
+    bail!("no client replied!")
```

Comment:
Was this supposed to say `No server replied`?

Reply:
`Couldn't get TO1 from any Rendezvous server?` maybe? Bear in mind that I want to implement retries after this PR.

Reply:
Right. I just figured the exactly message might be a bit off, but this is just a nitpick.

Reply:
I think it can be more descriptive too. WDYT about `Couldn't get TO1 from any Rendezvous server?` @puiterwijk